wordpress-mobile/WordPress-Android
# Remove unused date block from stats

This PR removes the date block from the DWMY stats after removing the type from FluxC. We'll replace it with the Date selector toolbar on top of the screen.

To test:
* Go to DWMY stats
* Notice the date block is no longer visible
